docs/tech-reference/yum-plugins.rst

```diff
@@ -530,6 +530,13 @@ configuration values are optional.
  ``force_full``
  Boolean flag. If true, full re-sync is triggered.
 
+``require_signature``
+Requires that imported packages like RPM/DRPM/SRPM should be signed. Defaults to False.
+
+``allowed_keys``
+List of allowed signature keys that imported packages can be signed with.
```

Comment:
Comma-separated I assume? That would be good to mention. Also it would be helpful to say that it should be the 8 character abbreviated fingerprint. Does capitalization matter?